Engines/Wine/Verbs/vcrun2003/script.js

```diff
@@ -22,29 +23,29 @@ Wine.prototype.vcrun2003 = function () {
         "msvcp71",
         "mfc71"
     ];
+
     var that = this;
```

Comment:
It seems like the verb scripts contain this assignment alot. 
@plata @ImperatorS79 do you know if this is still necessary with GraalVM?

Reply:
Normally yes, because without that you cannot access `wine` methods inside the `foreach`, however this is worth testing (just install the verb in dummy container to see).

Reply:
Ah I remember reading something about differences with the scope when using `function(args) {...}` compared to `(args) => {...}`. 

If I remember correctly the first variant automatically sets the scope inside the function to `this` (e.g. prototype constructors), while the second one does not.